misc/types.json

```diff
@@ -139,7 +139,8 @@
     "_enum": {
       "Authorized": "AccountId",
       "Court": null,
-      "SimpleDisputes": null
+      "SimpleDisputes": null,
+      "GlobalDisputes": null
```

Comment:
Don't forget to notice the new `GlobalDisputes` field in the changelog for devs file.